### PR TITLE
Bookstore create and remove

### PR DESCRIPTION
**Pull request by 1GORDON**
In this project, I achieved the following

- Configured my Redux Store.
- Added some UI elements to your components and made them dispatch actions.
- Wrote actions and reducer for adding and removing books.
- Configured the Redux Store.
- Installed redux logger.
- Set the data inputs in the local React state.
- Used the redux-logger to check if the application is working correctly through the browser's console.
- Live Link [bookstore-create-and-remove.netlify.app](https://bookstore-create-and-remove.netlify.app/)

### CODE REVIEW

File: src/components/CategoriesPage.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line
```
Comment (bimbolabuari) [inline src/components/CategoriesPage.js]: Kindly remove these disabled Linters, Linting helps in writing clean and error-free code ✊ 

Please check for other occurrences 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @1GORDON , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Configure the Redux Store `(/src/redux/configureStore.js)` and importing the necessary methods from Redux  👍 
- Deployed webpage 👍 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/CategoriesPage.js (hunk 1 above)
Comment (1GORDON) [reply src/components/CategoriesPage.js]: Fixed required changes

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Godanah, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**